src/content/samples/node.mdx

```diff
@@ -115,16 +112,17 @@ Welcome to your development container. Happy coding!
 cindy:hello-world app>
 ```
 
+> - Test your application end to end as fast as you type code
```

Comment:
why these changes here?

Reply:
Consistently, it was in the golang docs that you added L213

Reply:
But not here. They are in the `Next steps` section